core/src/main/java/org/wildfly/extras/creaper/core/online/OnlineManagementClientImpl.java

```diff
@@ -149,7 +149,8 @@ public void apply(Iterable<OnlineCommand> commands) throws CommandFailedExceptio
     public ModelNodeResult execute(ModelNode operation) throws IOException {
         checkClosed();
         operation = adjustOperationForDomain.adjust(operation);
-        log.debugf("Executing operation %s", ModelNodeOperationToCliString.convert(operation));
+        log.debugf("Executing operation %s\nJSON format:\n%s", ModelNodeOperationToCliString.convert(operation),
+                operation.toJSONString(false));
```

Comment:
This should absolutely be added to `execute(Operation)` too, yes. There's no way to do that for `execute(String)`, though.

One idea: I'm not terribly happy about newlines in a log message, how about adding it as a second log, possibly to the TRACE level?